src/orquestra/sdk/_base/cli/_dorq/_repos.py

```diff
@@ -502,3 +507,49 @@ def get_workflow_def(self, module, name: str) -> sdk.WorkflowDef:
         except exceptions.WorkflowSyntaxError:
             # Explicit re-raise
             raise
+
+
+class TokenRepo:
+    """A web server for automatically logging in a user"""
+
+    def __init__(
+        self,
+        runtime_repo: RuntimeRepo,
+        presenter: _presenters.LoginPresenter,
+        login_server: t.Optional[LoginServer] = None,
+    ):
+        self._runtime_repo = runtime_repo
+        self._presenter = presenter
```

Comment:
**issue**: can't see this being used anywhere. Is it needed?

Marking as blocker for the sake of separation of concerns in the "data" vs "presentation" layers.